tests/integ/test_packaging.py

```diff
@@ -326,12 +329,44 @@ def test_add_requirements_twice_should_fail_if_packages_are_different(
 
 
 @pytest.mark.skipif(
-    IS_IN_STORED_PROC,
-    reason="Subprocess calls are not allowed within stored procedures",
+    IS_IN_STORED_PROC or IS_WINDOWS,
+    reason="Subprocess calls are not allowed within stored procedures. Custom package upload does not work well on Windows.",
+)
+def test_add_unsupported_requirements_should_fail_if_custom_packages_upload_enabled_not_switched_on(
+    session, resources_path
+):
+    test_files = TestFiles(resources_path)
+    with patch.object(session, "_is_anaconda_terms_acknowledged", lambda: True):
+        with pytest.raises(
+            RuntimeError,
+            match="parameter 'custom_packages_upload_enabled' is set to False",
+        ):
+            session.add_requirements(test_files.test_unsupported_requirements_file)
+
+
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC or IS_WINDOWS,
+    reason="Subprocess calls are not allowed within stored procedures. Custom package upload does not work well on Windows.",
```

Comment:
Is there anything more specific to say (here and elsewhere) about why this doesn't work well on Windows?

Reply:
This is being investigated (https://github.com/snowflakedb/snowpark-python/pull/963), a concrete root cause has not been found yet.